### DISCUSSION TITLE
How to remove/uninstall cockpit-machines?

### DISCUSSION THREAD

**Discussion by landcraft** [cockpit-project/cockpit-machines#914] [Q&A] [answered]
I've decided to go a different way with my VM, but I got as far as installing the plugin in cockpit, so I now want it removed.
How do I do this?

**Comment by KKoukiou** [comment]
In the application page there is a remove button.
![Screen Shot 2023-01-13 at 18 17 49](https://user-images.githubusercontent.com/14921356/212379929-26221b90-2e49-4a8c-85cb-a6df977fcb3f.png)

**Reply by landcraft** [reply]
unfortunately it's not in there

**Reply by KKoukiou** [reply]
This is weird. But in any case then you can remove it from your distros package manager. In fedora for example, 'sudo dnf remove cockpit-machines'

**Reply by landcraft** [reply]
Actually I resolved the initial issue I had - got cockpit-machines up & running properly now.

**Reply by garrett** [reply]
@landcraft: What was your issue and how did you fix it? (We could try to make sure others don't hit the same issue, if it's not completely specific to your server.)

**Comment by landcraft** [comment, marked as answer]
I should point out that I resolved my initial problem by apt installing, rather than following the manual process on the front page.
